ChangeLog.md

```diff
@@ -20,6 +20,10 @@ See docs/process.md for more on how version tagging works.
 
 2.0.26
 ------
+- The alignment of `long double`, which is a 128-bit floating-point value
+  implemented in software, is reduced from 16 to 8. The lower alignment allows
+  `max_align_t` to be correctly defined as 8, and avoids raising it to 16 which
```

Comment:
I would rephrase `to be correctly defined as 8` to something more like `to match our the alignment we use for malloc`.    `max_align_t` was never incorrect as such.. if anything out malloc implementations were incorrect for not honoring `max_align_t` in the first place.